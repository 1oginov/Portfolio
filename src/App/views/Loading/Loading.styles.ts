import { withStyles } from '@material-ui/core';

export default withStyles({
  root: {
    alignItems: 'center',
    display: 'flex',
    flexDirection: 'column',
    flexGrow: 1,
    justifyContent: 'center',
  },
});
