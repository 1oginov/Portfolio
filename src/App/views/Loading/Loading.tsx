import * as React from 'react';

import Progress from 'Shared/components/Progress';

const Loading: React.FunctionComponent = () => (
  <React.Fragment>

    <h1>Loading</h1>

    <div>
      <Progress />
    </div>

  </React.Fragment>
);

export default Loading;
