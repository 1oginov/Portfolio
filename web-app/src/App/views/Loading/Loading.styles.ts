import { withStyles } from '@material-ui/core';

import { viewRoot } from 'Styles/mixins';

export default withStyles(theme => ({
  root: {
    ...viewRoot(theme),
    justifyContent: 'center',
  },
}));
