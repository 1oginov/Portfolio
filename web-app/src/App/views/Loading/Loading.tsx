import * as React from 'react';

import Progress from 'Shared/components/Progress';

interface Props {
  classes: { [key: string]: string };
}

const Loading: React.FunctionComponent<Props> = ({ classes }: Props) => (
  <div className={classes.root}>
    <Progress size="large" />
  </div>
);

export default Loading;
