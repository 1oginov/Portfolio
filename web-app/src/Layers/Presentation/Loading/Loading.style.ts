import { withStyles } from '@material-ui/core';

import { viewRoot } from 'Styles/mixins';

export const style = withStyles((theme) => ({
  root: {
    ...viewRoot(theme),
    justifyContent: 'center',
  },
}));
