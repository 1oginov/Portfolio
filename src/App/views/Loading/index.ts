import Loading from './Loading';
import style from './Loading.styles';

export default style(Loading);
