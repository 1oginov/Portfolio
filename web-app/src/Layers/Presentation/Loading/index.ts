import { Loading as LoadingComponent } from './Loading';
import { style } from './Loading.style';

export const Loading = style(LoadingComponent);
