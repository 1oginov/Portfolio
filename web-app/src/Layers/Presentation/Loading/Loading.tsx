import * as React from 'react';

import { Progress } from 'Layers/Presentation/Progress';

interface Props {
  classes: { [key: string]: string };
}

export const Loading: React.FunctionComponent<Props> = ({ classes }: Props) => (
  <div className={classes.root}>
    <Progress size="large" />
  </div>
);
