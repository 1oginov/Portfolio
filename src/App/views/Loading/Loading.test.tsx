/* eslint-disable import/no-extraneous-dependencies */

import { shallow } from 'enzyme';
import classes from 'identity-obj-proxy';
import * as React from 'react';

import Loading from './Loading';

it('matches snapshot', () => {
  const wrapper = shallow(<Loading classes={classes} />);

  expect(wrapper).toMatchSnapshot();
});
