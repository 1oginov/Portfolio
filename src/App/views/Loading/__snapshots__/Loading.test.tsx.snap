// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Fragment>
  <h1>
    Loading
  </h1>
  <div>
    <Progress />
  </div>
</Fragment>
`;
